Dockerfile

```diff
@@ -14,6 +14,9 @@ RUN sed -i '/fuzz/d' Cargo.toml && \
     sed -i '/test\_/d' Cargo.toml && \
     sed -i '/profiler/d' Cargo.toml
 
+# Use Makefile to build
+COPY Makefile ./
```

Comment:
Should we pick https://github.com/tikv/tikv/pull/5017?

Reply:
Nice catch